tests/index.ts

```diff
@@ -0,0 +1,7 @@
+import 'reflect-metadata';
+import { useContainer as routingUseContainer } from 'routing-controllers';
+import { useContainer as ormUseContainer } from 'typeorm';
+import Container from 'typedi';
+
+routingUseContainer(Container);
+ormUseContainer(Container);
```

Comment:
Is this all necessary if we're manually instantiating everything through constructors?

Reply:
Don't think it's necessary anymore